<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="" lang="">
<head>
  <title>Test Results Report - PerOffice2</title>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
  <meta name="description" content="TestNG unit test results." />
  <link href="reportng.css" rel="stylesheet" type="text/css" />
    <script type="text/javascript" src="reportng.js"></script>
</head>
<body>
<h1>PerOffice2</h1>
<p>
  Test duration: 471.440s
</p>



  <table class="resultsTable">
    <tr><th colspan="3" class="header failed">Failed Tests</th></tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.MassMessage_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  testMassMsg
      </td>
  <td class="duration">
    15.499s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-0', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: expected [true] but found [false]</b></a><br />
    <div class="stackTrace" id="exception-0">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertTrue(Assert.java:44)<br />
              org.testng.Assert.assertTrue(Assert.java:54)<br />
              com.webtest.PerOffice2.demo.MassMessage_Test.testMassMsg(MassMessage_Test.java:20)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:580)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.PerInform_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  test2
      </td>
  <td class="duration">
    9.679s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-1', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: expected [true] but found [false]</b></a><br />
    <div class="stackTrace" id="exception-1">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertTrue(Assert.java:44)<br />
              org.testng.Assert.assertTrue(Assert.java:54)<br />
              com.webtest.PerOffice2.demo.PerInform_Test.test2(PerInform_Test.java:27)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:580)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.PerPwd_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  test_failed1
      </td>
  <td class="duration">
    13.033s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-2', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: expected [true] but found [false]</b></a><br />
    <div class="stackTrace" id="exception-2">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertTrue(Assert.java:44)<br />
              org.testng.Assert.assertTrue(Assert.java:54)<br />
              com.webtest.PerOffice2.demo.PerPwd_Test.test_failed1(PerPwd_Test.java:19)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:580)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
<tr>
  <td class="method">
                  test_failed2
      </td>
  <td class="duration">
    0.078s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-3', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: expected [true] but found [false]</b></a><br />
    <div class="stackTrace" id="exception-3">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertTrue(Assert.java:44)<br />
              org.testng.Assert.assertTrue(Assert.java:54)<br />
              com.webtest.PerOffice2.demo.PerPwd_Test.test_failed2(PerPwd_Test.java:27)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:580)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
<tr>
  <td class="method">
                  test_failed3
      </td>
  <td class="duration">
    0.078s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-4', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: expected [true] but found [false]</b></a><br />
    <div class="stackTrace" id="exception-4">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertTrue(Assert.java:44)<br />
              org.testng.Assert.assertTrue(Assert.java:54)<br />
              com.webtest.PerOffice2.demo.PerPwd_Test.test_failed3(PerPwd_Test.java:35)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:580)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
<tr>
  <td class="method">
                  test_failed4
      </td>
  <td class="duration">
    0.078s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-5', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: expected [true] but found [false]</b></a><br />
    <div class="stackTrace" id="exception-5">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertTrue(Assert.java:44)<br />
              org.testng.Assert.assertTrue(Assert.java:54)<br />
              com.webtest.PerOffice2.demo.PerPwd_Test.test_failed4(PerPwd_Test.java:43)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:580)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
<tr>
  <td class="method">
                  test_failed5
      </td>
  <td class="duration">
    0.078s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-6', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: expected [true] but found [false]</b></a><br />
    <div class="stackTrace" id="exception-6">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertTrue(Assert.java:44)<br />
              org.testng.Assert.assertTrue(Assert.java:54)<br />
              com.webtest.PerOffice2.demo.PerPwd_Test.test_failed5(PerPwd_Test.java:51)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:580)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.TelMessage_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  testdelfailed
      </td>
  <td class="duration">
    12.102s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-7', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: expected [true] but found [false]</b></a><br />
    <div class="stackTrace" id="exception-7">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertTrue(Assert.java:44)<br />
              org.testng.Assert.assertTrue(Assert.java:54)<br />
              com.webtest.PerOffice2.demo.TelMessage_Test.testdelfailed(TelMessage_Test.java:19)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:580)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.WordLimit_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  testWordLimit1
      </td>
  <td class="duration">
    20.391s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-8', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: expected [false] but found [true]</b></a><br />
    <div class="stackTrace" id="exception-8">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertFalse(Assert.java:65)<br />
              org.testng.Assert.assertFalse(Assert.java:75)<br />
              com.webtest.PerOffice2.demo.WordLimit_Test.testWordLimit1(WordLimit_Test.java:21)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:580)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
<tr>
  <td class="method">
                  testWordLimit2
      </td>
  <td class="duration">
    18.265s
  </td>
  <td class="result">
        
    
        

        <a href="javascript:toggleElement('exception-9', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: expected [true] but found [false]</b></a><br />
    <div class="stackTrace" id="exception-9">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertTrue(Assert.java:44)<br />
              org.testng.Assert.assertTrue(Assert.java:54)<br />
              com.webtest.PerOffice2.demo.WordLimit_Test.testWordLimit2(WordLimit_Test.java:31)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.base/java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:580)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
      </table>


  <table class="resultsTable">
    <tr><th colspan="3" class="header passed">Passed Tests</th></tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.AddReadTime_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  testReadTime
      </td>
  <td class="duration">
    14.750s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.Comment_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  testComment
      </td>
  <td class="duration">
    18.202s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.DelPerLog_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  testDelPerLog
      </td>
  <td class="duration">
    13.449s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.FindInform_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  test_findbyright
      </td>
  <td class="duration">
    15.135s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.FindTel_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  test_findbykey
      </td>
  <td class="duration">
    17.781s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.NewLog_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  testNewLog
      </td>
  <td class="duration">
    15.078s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.NewTel_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  test_noinfo
      </td>
  <td class="duration">
    22.824s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.PerInform_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  test1
      </td>
  <td class="duration">
    13.523s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
<tr>
  <td class="method">
                  test3
      </td>
  <td class="duration">
    14.660s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
<tr>
  <td class="method">
                  test4
      </td>
  <td class="duration">
    14.383s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.webtest.PerOffice2.demo.ReadShortMsg_Test</td>
      </tr>
            
<tr>
  <td class="method">
                  testReadShortMsg
      </td>
  <td class="duration">
    13.308s
  </td>
  <td class="result">
        
    
        

      </td>
</tr>
      </table>
  
</body>
</html>
